packages/design-tokens/tokens/motion-ease.yml

```diff
@@ -0,0 +1,10 @@
+global:
+  category: motion-ease
+  type: motion-ease
+props:
+  - name: motion-ease-in
+    value: 'cubic-bezier(.35, 0, .7, .2)'
+  - name: motion-ease-out
+    value: 'cubic-bezier(0, 0, .3, 1)'
+  - name: motion-ease-inout
```

Comment:
This is a tad picky, but the baked-in CSS timing function is called `ease-in-out` (rather than `ease-inout`). What would you think about making that consistent by naming this `motion-ease-in-out`?